docs/image-build.md

```diff
@@ -0,0 +1,24 @@
+
+# Kube-compare image build
+
+## Build container
+
+```bash
+make image-build
+```
+
+## Copy binary locally
+
+- One option it to build the image locally
```

Comment:
I think this should be binary not image.

Reply:
Done